Makefile

```diff
@@ -18,15 +18,15 @@ ifeq ($(UNAME), Darwin)
 	# do something OSX
 	CXX = clang++ -arch x86_64 -arch arm64
 	EXT = dylib
-	XLD_FLAGS = -arch x86_64 -arch arm64
+	XLDFLAGS =
```

Comment:
Curious to know whether any of the `XLDFLAGS` are actually required?

Reply:
For some of the compilers they are required (I think!)